neurodamus/io/sonata_config.py

```diff
@@ -371,28 +369,21 @@ def parsedStimuli(self):
         }
         module_translation = {"seclamp": "SEClamp", "subthreshold": "SubThreshold"}
 
-        stimuli = {}
-        for name in self._sim_conf.list_input_names:
+        stimuli = []
+        # TODO: loop over self._sim_conf.list_input_names,
+        # The order of stimulus injection could lead to minor difference on the results
+        # so need to preserve it as in the config file, libsonata API
+        for name in self._sections["inputs"]:
             stimulus = self._translate_dict("inputs", self._sim_conf.input(name))
```

Comment:
while you are at it, may you add a docstring at this function too?

Reply:
which function do you mean in this comment?

Reply:
`_translate_dict`

Reply:
Done